docs/docs/benchmark.md

```diff
@@ -1,96 +1,164 @@
-# Benchmarking
+# Benchmarking suites
 
-<p>OpenML significantly simplifies benchmarking by:</p>
-</li><li>offering a large set of datasets in standardized data formats
-</li><li>easily download through a wide range of [APIs](APIs) and existing client libraries</a>
-</li><li>machine-readable meta-information on all the datasets
-</li><li>sharing of benchmarking results in a reproducible way through the [APIs](APIs), enabling large scale comparisons
-</li></ul>
-
-In addition, OpenML offers <b>benchmarking suites</b>: curated, comprehensive sets of machine learning datasets,
+OpenML offers <b>benchmarking suites</b>: curated, comprehensive sets of machine learning datasets,
```

Comment:
Not all benchmark suites are necessarily curated though. The OpenML100 and CC18 are.